From 5ac0ad60b79a7e421defa04f846a57b46c3435b6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Thu, 5 May 2022 10:28:45 +0100
Subject: [PATCH] vertical text test fails with latest harfbuzz

with font "Droid Sans Japanese" which doesn't seem to be available in
ci, but is by default in the Fedora 37 build roots

I suspect since: https://github.com/harfbuzz/harfbuzz/issues/537

Change-Id: I23d9415444668ce379765f32a96ed67f057ebfd7
---
 vcl/qa/cppunit/text.cxx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/vcl/qa/cppunit/text.cxx b/vcl/qa/cppunit/text.cxx
index 5ab706045606..899217347085 100644
--- a/vcl/qa/cppunit/text.cxx
+++ b/vcl/qa/cppunit/text.cxx
@@ -315,7 +315,7 @@ void VclTextTest::testVerticalText()
     exportDevice("vertical-text-36.png", device);
     // Height of U+30E8 with font 36 size should be roughly 28 pixels,
     // but since we don't know which font will be used, allow even more range.
-    tools::Long height36 = getCharacterRightSideHeight(device, Point(99, 20));
+    tools::Long height36 = getCharacterRightSideHeight(device, Point(99, 22));
     CPPUNIT_ASSERT_DOUBLES_EQUAL(tools::Long(28), height36, 6);
     tools::Long width36 = getCharacterTopWidth(device, Point(65, 0));
     CPPUNIT_ASSERT_DOUBLES_EQUAL(tools::Long(25), width36, 6);
@@ -390,7 +390,7 @@ void VclTextTest::testVerticalText()
     device->SetFont(font);
     device->DrawText(Point(90, 10), text);
     exportDevice("vertical-text-36-50pct.png", device);
-    tools::Long height36pct50 = getCharacterRightSideHeight(device, Point(99, 15));
+    tools::Long height36pct50 = getCharacterRightSideHeight(device, Point(99, 16));
     CPPUNIT_ASSERT_DOUBLES_EQUAL(height36 / 2, height36pct50, 2);
     tools::Long width36pct50 = getCharacterTopWidth(device, Point(65, 0));
     CPPUNIT_ASSERT_DOUBLES_EQUAL(width36, width36pct50, 2);
-- 
2.35.1

